.github/workflows/docs.yaml

```diff
@@ -48,7 +48,9 @@ jobs:
         run: yarn lint:fix
 
       - name: Move OpenAPI spec file into documentation directory
-        run: mv packages/hoprd/rest-api-v2-full-spec.yaml docs/hopr-documentation/
+        run: |
+          mv packages/hoprd/rest-api-v2-full-spec.json docs/hopr-documentation/
+          git add docs/hopr-documentation/rest-api-v2-full-spec.json
 
       - name: Commit docs changes
         # only commit changes when not running via act and when on final branch<Plug>_es
```

Comment:
Why has the docusaurus configuration file been removed?

Reply:
must be a merging issue, fixing

Reply:
added back, please check & resolve